### PR TITLE
(WIP) 412 Community spec license adoption

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

File: NOTICES
Comment (robmoffat) [inline NOTICES:12]: We've removed a section here since the procedure for signing up doesn't match what we actually do.

Can someone Legal review?

Comment (robmoffat) [reply NOTICES:12]: https://github.com/finos/FDC3/blob/master/README.md#how-do-you-become-a-participant-editor-or-maintainer

Comment (robmoffat) [reply NOTICES:12]: @mindthegab can you add a legal reviewer?

Comment (robmoffat) [reply NOTICES:12]: Currently, people send an email to fdc3-participants+subscribe@finos.org

containing something like:

> HI, my name is <FirstName LastName> and I'd like to formally participate to the FDC3 standard process. I plan to contribute as <individual|on behalf of organizationName> and I have reviewed the policies described at https://github.com/finos/community/tree/master/governance/Standards-Projects . Thank you!

**Comment by robmoffat** [comment]
@kriswest is there a way of kicking off the preview here?

**Comment by kriswest** [comment]
@robmoffat you have to touch a file in the /website folder to trigger a preview. Yes its a little annoying, I did submit a support query to FINOS/LF who were supposed to be having a think about what could be done about it...

**Comment by kriswest** [comment]
@robmoffat preview is processing now

**Comment by robmoffat** [comment]
awesome thanks Kris

**Review by nkolba — APPROVED** [review]
LGTM

**Comment by robmoffat** [comment]
@scooter99boston:

The FDC3 ICLA is currently "Fintech Open Source Foundation Individual Contributor License Agreement"
The CCLA is: "Fintech Open Source Foundation Corporate Contributor License Agreement"
The current CLA agreements are setup at the parent FINOS Foundation level - meaning, the CLA agreements apply to all child projects (such as FDC3).

You need to decide: are these CLA/CCLAs compatible with our change to Community Specification License, or, do we need to set an override of the ICLA/CCLA for this project and get contributors to sign a new agreement?

@dealako

**Comment by robmoffat** [comment]
@jgavronsky

**Comment by robmoffat** [comment]
> , or, do we need to set an override of the ICLA/CCLA for this project and get contributors to sign a new agreement?

@scooter99boston says this is what we need to do.  

We need to use the CSL CLA.

**Review by scooter99boston — APPROVED** [review]

**Comment by robmoffat** [comment]
@mindthegab please can you review, especially the README please.

File: README.md
Comment (mindthegab) [inline README.md:41]: @robmoffat should we link here to https://fdc3.finos.org/community? Something like: 

"For a list of open source and proprietary desktop agents, see the "Platform providers" [here](https://fdc3.finos.org/community#type-platform-provider)"

Comment (mindthegab) [inline README.md:41]: @robmoffat should we link here to https://fdc3.finos.org/community ?  Something like:

"For a list of open source and proprietary desktop agents, see "Platform providers" [here](https://fdc3.finos.org/community#type-platform-provider)"

File: README.md
Comment (mindthegab) [inline README.md:80]: @robmoffat Shall we add at the end:

"And Open Source Electron-based implementation of an FDC3 Desktop Agent is available [here](https://github.com/finos/electron-fdc3)".

File: README.md
Comment (mindthegab) [inline README.md:88]: @robmoffat should we mention this page at all https://www.finos.org/get-involved-fdc3 (aka you plan to keep it updated?) 

Or do you want to direct people to directly raise a PR against the community page? 

Your call.

File: README.md
Comment (mindthegab) [inline README.md:133]: @robmoffat @scooter99boston shouldn't this be more specific and talk about the CSL CLA? (vs the generic FINOS ICLA).

Should we also link to the Contributor_License_Agreement.md file?

File: README.md
Comment (mindthegab) [inline README.md:164]: @robmoffat @scooter99boston again, shouldn't this be more specific that it's not a FINOS CLA, rather a CSL CLA that will be signed?

In other words, if they have a FINOS (Apache) CCLA/ICLA signed, it will have no bearing on FDC3 and they need to follow the procedure above. 

I think this needs to be crystal clear. Maybe add a specific NOTE on it.

File: README.md
Comment (mindthegab) [inline README.md:156]: @robmoffat  I would amend with:

"If you are going to participate on an ongoing basis, use, implement or benefit from the FDC3 standard..."

File: LICENSE.md
Comment (mindthegab) [inline LICENSE.md:20]: @robmoffat we're missing a reference to the 2.0 version of the standard. 

@scooter99boston I think it should say "1.1, 1.2 and 2.0 are subject to the FINOS IP Policy"

File: README.md
Comment (mindthegab) [inline README.md:179]: @robmoffat 2 points here:

1. Always link to the FINOS downstream fork https://github.com/finos/standards-project-blueprint (as there are changes like default apache license), not to the upstream CSL
3. I'd be more explicit here and say: "FDC3 specifications in the repository (AFTER Version 2.0) are subject to the FINOS **Community Specification License 1.0** available at [https://github.com/finos/standards-project-blueprint].  For versions up to 2.0 and more details see the [license](LICENSE.md).

File: README.md
Comment (mindthegab) [inline README.md:66]: @robmoffat should we also link the [FDC3 znglist](https://resources.finos.org/category/projects-sigs/fdc3/?projects-sigs=fdc3) where marketing consolidate educational material for FDC3 users?

File: LICENSE.md
Comment (mindthegab) [inline LICENSE.md:5]: I think we need to link to the

[FINOS Community Specification](https://github.com/finos/standards-project-blueprint/)

which is indeed the copy, with minor / non breaking changes (to the governance, not to the license) that the board approved as technical charter (@scooter99boston).

Our goal is to keep it in line with the upstream CSL, so we can keep calling it CSL, but for clarity I think it's important to link the appropriate FINOS downstream copy (which has for example the Apache source code as default provision in it).

**Review by mindthegab — CHANGES_REQUESTED** [review]
Hey @robmoffat @scooter99boston - looks good - done with this review, and asked for a few changes.

File: README.md
Comment (robmoffat) [reply README.md:41]: done

File: README.md
Comment (robmoffat) [reply README.md]: I've linked it in the community page, and referenced the community page here.  There's way more information on the community page than in the znglist atm.

File: README.md
Comment (robmoffat) [reply README.md:85]: now added in the resources section, above 👍

File: README.md
Comment (robmoffat) [reply README.md:93]: I think I'd rather get the email.   I've ask Niamh to take that page down - it's out-of-date and contained many of the wrong questions anyway.

File: README.md
Comment (robmoffat) [reply README.md:138]: HI Gab,  

I'd have to get @scooter99boston's opinion via email on this if you want it.  However, I'd like to point out that even though this is marked in green on the diff, this is exactly the same copy from the existing README.md file.  

Although it's not specific to the CSL CLA, I think it's correct in pointing out that contributors are going to have to tangle with EasyCLA, which seems to be the basic point.  

I'd like to leave it but your call.

File: LICENSE.md
Comment (robmoffat) [reply LICENSE.md:20]: @kriswest can you comment on this?

File: README.md
Comment (robmoffat) [reply README.md:169]: yes, I think this needs to change.  At the moment, I still don't have the PDFs for the ICLA or CCLA under CSL.  I'll link these when I get them added on EasyCLA.

Comment (robmoffat) [reply README.md]: Ok, I've tidied this up and linked to the EasyCLA docs too.  Added your note.

File: LICENSE.md
Comment (kriswest) [reply LICENSE.md:20]: They are correct - 1.1, 1.2 and 2.0 are all subject to the FINOS IP policy

**Pull request closed without merging**
